releases/1.7/edge/kubeflow/bundle.yaml

```diff
@@ -55,15 +55,16 @@ applications:
     scale: 1
     _github_repo_name: katib-operators
   katib-db:
-    charm: charmed-osm-mariadb-k8s
-    channel: latest/stable
+    charm: mysql-k8s
+    channel: 8.0/stable
     scale: 1
     options:
       database: katib
```

Comment:
We are using the mysql-k8s charm, which does not have these options. Same applies to kfp-db.
```suggestion
```